src/documentation/components/card/card.css

```diff
@@ -0,0 +1,135 @@
+@import "../../../index.css";
+@import "https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css";
+
+.checked {
+  color: var(--primary);
+}
+
+
+.card-rating {
+  margin: 0.1rem 0rem 0.2rem 0rem;
+}
+
+.card {
+  padding: 1.5rem 1.5rem;
+  background-color: var(--white);
+
+  width: 13rem;
+  border-radius: 5px;
+  display: flex;
+  flex-direction: column;
+  justify-content: space-between;
+  height: 21rem;
+
+  box-shadow: 5px 10px 10px black;
+  position: relative;
+  border:1px solid #DCDCDC
+}
+.out-of-stock {
+  position: absolute;
+  background-color: rgba(0, 0, 0, 0.6);
+  top: 0;
+  left: 0;
+  right: 0;
+  bottom: 0;
+  z-index: 3;
+  text-align: center;
+}
+
+.out-of-stock > p {
+  padding-top: 8rem;
+  font-size: 2rem;
+  color: var(--primary);
+}
+.card-img-container {
+  display: flex;
+  width: 100%;
+  justify-content: center;
+  position: relative;
+  height: 12rem;
+
+  margin-bottom: 0.5rem;
+  border: 1px solid #e2ebef;
```

Comment:
1. The color used could have been a variable from the color palette of your making